clients/unshaded/src/main/java/tachyon/hadoop/AbstractTFS.java

```diff
@@ -88,22 +85,7 @@
   public FSDataOutputStream append(Path cPath, int bufferSize, Progressable progress)
       throws IOException {
     LOG.info("append({}, {}, {})", cPath, bufferSize, progress);
-    if (mStatistics != null) {
-      mStatistics.incrementWriteOps(1);
-    }
-    TachyonURI path = new TachyonURI(Utils.getPathWithoutScheme(cPath));
-    TachyonFile file;
-    try {
-      file = mTFS.open(path);
-      if (mTFS.getInfo(file).length > 0) {
-        LOG.warn("Appending to nonempty file. This may be an error.");
-      }
-    } catch (TachyonException e) {
-      throw new IOException(e);
-    }
-
-    return new FSDataOutputStream(mTFS.getOutStream(file.getFileId(), OutStreamOptions.defaults()),
-        mStatistics);
+    throw new UnsupportedOperationException("Append is not supported in Tachyon.");
```

Comment:
This changes the existing behavior. The old behavior checks whether the file exists or not, and if not treats it like creating a new file.